.travis.yml

```diff
@@ -47,67 +48,113 @@ matrix:
+          env: PYTHON_VERSION=2.7 SETUP_CMD='test -V'
+               CONDA_DEPENDENCIES='Cython click'
+               PIP_DEPENDENCIES=''
         - os: linux
-          env: PYTHON_VERSION=3.4 OPTIONAL_DEPS=false SETUP_CMD='test -V'
-
-        # Test with other numpy versions
+          env: PYTHON_VERSION=3.4 SETUP_CMD='test -V'
+               CONDA_DEPENDENCIES='Cython click'
+               PIP_DEPENDENCIES=''
+
+        # Test with other numpy versions. Not all of the packages are
+        # available with these on conda, thus moving some of them to pip
+        # install. Latest astropy stable (1.1) is also not available for
+        # older numpies, but using lts version should be good enough for
+        # these tests (so we can avoid building it from source).
         - os: linux
           env: PYTHON_VERSION=3.4 NUMPY_VERSION=dev SETUP_CMD='test -V'
         - os: linux
-          env: PYTHON_VERSION=2.7 NUMPY_VERSION=1.8 SETUP_CMD='test -V'
-        - os: linux
-          env: PYTHON_VERSION=2.7 NUMPY_VERSION=1.7 SETUP_CMD='test -V'
+          env: PYTHON_VERSION=2.7 NUMPY_VERSION=1.9 SETUP_CMD='test -V'
+               SHERPA=true ASTROPY_VERSION=lts
         - os: linux
-          env: PYTHON_VERSION=2.7 NUMPY_VERSION=1.6 SETUP_CMD='test -V'
+          env: PYTHON_VERSION=2.7 NUMPY_VERSION=1.8 SETUP_CMD='test -V'
+               SHERPA=true ASTROPY_VERSION=lts
+               CONDA_CHANNELS='astropy-ci-extras astropy'
+               CONDA_DEPENDENCIES='Cython click matplotlib'
+               PIP_DEPENDENCIES=''
 
     # You can move builds that temporarily fail because of some non-Gammapy issue here
     # Please add a link to a GH issue that tracks the upstream issue.
     allow_failures:
       - env: PYTHON_VERSION=3.4 NUMPY_VERSION=dev SETUP_CMD='test -V'
-      # Tests for old Numpy versions currently don't work:
-      # https://github.com/astropy/conda-builder-affiliated/issues/40
-      # - env: PYTHON_VERSION=2.7 NUMPY_VERSION=1.8 SETUP_CMD='test -V'
       - env: PYTHON_VERSION=2.7 NUMPY_VERSION=1.7 SETUP_CMD='test -V'
-      # - env: PYTHON_VERSION=2.7 NUMPY_VERSION=1.6 SETUP_CMD='test -V'
+             SHERPA=true
+             CONDA_CHANNELS='astropy-ci-extras astropy'
 
 install:
-    - source .continuous-integration/travis/setup_environment_$TRAVIS_OS_NAME.sh
+    - git clone git://github.com/astropy/ci-helpers.git
+    - source ci-helpers/travis/setup_conda_$TRAVIS_OS_NAME.sh
+
+    - if $FETCH_GAMMAPY_EXTRA; then
+          git clone https://github.com/gammapy/gammapy-extra.git $HOME/gammapy-extra;
+          export GAMMAPY_EXTRA=${HOME}/gammapy-extra;
+      fi
+
+    # This is needed to make matplotlib plot testing work
+    - if [[ $TRAVIS_OS_NAME == 'linux' ]]; then
+          export DISPLAY=:99.0;
+          sh -e /etc/init.d/xvfb start;
+          export QT_API=pyqt;
+      fi
+
+    # Reinstalling the required numpy version as it may be downgraded
```

Comment:
yes, I've added it after seeing travis fails because of messed up np versions. However that was still on the v4.7 branch. I'll open an issue for reminding to revise this part once the 4.8 conda packages become available.

E.g.: https://travis-ci.org/gammapy/gammapy/jobs/99867950#L639